### PR TITLE
Added information about use of Gherkin

### LINKED ISSUE #18: Add information about use of Gherkin
**Description**
We're using Gherkin more-and-more, but haven't documented that fact yet.

**Scope**
Add a description of our use of Gherkin to the Documentation page.

**Background and/or Other Information**
Some links we've been finding useful:
 - [BDD 101: Writing Good Gherkin](https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/)
 - [Behat: Writing Features](http://docs.behat.org/en/v2.5/guides/1.gherkin.html)

### PR DESCRIPTION
**Pull request by pbarabe**
Adds information about GradColl use of Gherkin and writing good features. Also expanded on the discussion about where and how to store project documentation more generally.

Also fixes a heading error in the CSS best practices file and updates a vulnerable package dependency.

Fixes #18

### CODE REVIEW

File: docs/best-practices/documentation.md [hunk 1]
```diff
@@ -1,40 +1,51 @@
...
 Developer-facing documentation is essential for communicating information about how to begin development on a new project, or how a project is designed to be configured and used.
 This best practice is designed to be a guide for repository-level documentation, i.e., the documentation which would live on the filesystem in the project's source control repository.
 
-## Specific Standards
 
-### README
+## Rationale
+
+Documentation preserves developer knowledge and reduces waste.
+It gives a developers a place to explain how to use their project to other developers and it helps new developers get up and running quickly.
```
Comment (uacate) [inline docs/best-practices/documentation.md]: Probably want to change "It gives a developers...." to "It gives a developer..." or "It gives developers..."

File: docs/best-practices/documentation.md [hunk 2]
```diff
@@ -1,40 +1,51 @@
...
 ```txt
 Copyright (c) 2019 The Arizona Board of Regents on behalf of the University of Arizona - All Rights Reserved.
 ```
 
-### Editorconfig
+#### Editorconfig
 
-Add an `.editorconfig` file. In most cases, you can use the [examples/.editorconfig](https://github.com/uazgraduatecollege/developer-essentials/blob/master/examples/.editorconfig) file.
+Add an `.editorconfig` file.
+In most cases, you can use the [examples/.editorconfig](https://github.com/uazgraduatecollege/developer-essentials/blob/master/examples/.editorconfig) file.
```
Comment (uacate) [inline docs/best-practices/documentation.md]: [editorconfig documentation](https://editorconfig.org/)

File: docs/best-practices/documentation.md (hunk 1 above)
Comment (pbarabe) [reply docs/best-practices/documentation.md]: @uacate Good catch! Updated.

File: docs/best-practices/documentation.md [hunk 3]
```diff
@@ -50,16 +61,33 @@ For projects hosted primarily in BitBucket, any of these templates may be copied
...
 ### Additional documentation
 
-Additional documentation should be located in `./docs`.
+Additional documentation should be located in `./docs/`.
 
-[Docsify](https://docsify.js.org/#/) is a nice static site generator for documentation.
+### Features
 
-See this repository's `docs/index.html` for a usage example.
+We're starting to use Gherkin for describing user-facing functionality, and have learning about best practices from a couple of resources:
```
Comment (uacate) [inline docs/best-practices/documentation.md]: "...describing user-facing functionality, and have learning about"
probably should have this be:
"...describing user-facing functionality, and we're learning..."

File: examples/README.md [hunk 4]
```diff
@@ -96,3 +95,15 @@ We use [SemVer](http://semver.org/) for versioning. For the versions available,
 Most Grad IT projects will belong to ABOR, so use the following (be sure to update the year as needed):
 
 Copyright (c) 2019 Arizona Board of Regents on behalf of the University of Arizona - All Rights Reserved.
+
+## Resources (optional)
+
+Resources can include things like websites that help to figure out how to do something.
```
Comment (uacate) [inline examples/README.md]: Resources can include things like websites that help to figure out how to do something

change to:

Resources can include things like websites that help you figure out how to do something

**Pull request merged**